src/components/play-page.js

```diff
@@ -122,6 +122,7 @@ export class FlashCards extends connect(store)(PageViewElement) {
       <a-card hidden?="${_showSettingsPage}"
         question="${_card.question}"
         answer="${_card.answer}"
```

Comment:
don't think you need to pass the answer too if you're passing the whole array (it's just duplicating data, and the `a-card` element should just use the first one in the display)